developers.diem.com/docs/node/overview.md

```diff
@@ -17,8 +17,8 @@ Learn how to start, configure, and monitor a Diem Node
     to="/docs/node/nodes#validator-nodes-validators"
   />
   <SimpleTextCard
-    icon="img/libra-nodes.svg"
-    iconDark="img/libra-nodes-dark.svg"
+    icon="img/diem-nodes.svg"
```

Comment:
We would need to rename the actual images too, right? 

They would be in `developers.diem.com/static/img`